queries.sql

```diff
@@ -12,4 +12,21 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 
 SELECT * FROM animals WHERE neutered = TRUE;SELECT * FROM animals WHERE neutered = TRUE;
 
- SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+SELECT COUNT(id) FROM animals;
```

Comment:
Kindly add all the queries that you have in your screenshots to the queries.sql file as mentioned in the project requirements 👍 👍

Reply:
Hi @Danieldotcomcoder. All the queries are added. Can you tell me which one isn't?

Reply:
Hello @Kamba56 , The queries in your screenshots are not present in the queries file

Reply:
@Danieldotcomcoder. Are you talking about the transactions?

Reply:
Please just tell me which one is not added because I have checked over and over again and all are added except the transactions because the Project requirement didn't demand me to add Transactions. @Danieldotcomcoder

Reply:
I am talking about these queries , they are also queries that needed to be inside the queries file, Its mentioned in the project requirement
![image](https://user-images.githubusercontent.com/87448628/185739219-d73f4390-08a5-42b2-8326-5c42daae07f7.png)
![image](https://user-images.githubusercontent.com/87448628/185739233-d9bda03b-e258-4866-92bd-e7f4b233aa49.png)
![image](https://user-images.githubusercontent.com/87448628/185739238-208a2105-7fb8-42b1-a46c-d1557903ddf1.png)
![image](https://user-images.githubusercontent.com/87448628/185739247-e899375c-66ef-420f-add4-f660f5c631b1.png)

Reply:
🤦🏾‍♂️ Like I said, the first three you mentioned are there in the queries.sql. I don't know how you can't see it, they are literally just below your comment in the files changed.

Reply:
@Danieldotcomcoder. Deleting from database is literally not a query, it's an update to the database. Queries are requests for data from the database.